### DISCUSSION TITLE
Cannot hear my bandmates and the other way round

### DISCUSSION THREAD

**Discussion by Schmidl83** [jamulussoftware/jamulus#2641] [Technical Support & Help]
Hi there,
I'm new to this forum and I hope you can help me.
Yesterday we were trying to do a jamulus session, but we all had the problem that we could barely hear each other. Only when I cranked my headphones to max, I could hear my bandmates a little. Everyone except one had the same issue.
Last year when I used jamulus the last time, we didn't have any issues of this kind at all.
I can hear myself perfectly.
I have my onw server set up. Connection works fine for everyone.
I have the latest ASIO driver and Jaumulus software installed.
I use a UMC 1820.

Is anybody familiar with this behavior?

Thank you!

BR,
Daniel

**Comment by Schmidl83** [comment]
![signal-2022-05-22-18-17-11-884](https://user-images.githubusercontent.com/106056039/169766827-a9b0e9bf-25f9-423d-b47c-ef9c499606f6.jpg)

Here are my client settings.

**Reply by pljones** [reply]
> I could hear my bandmates a little.

Could you make that clearer.  Do you mean the sound was simply _quiet_ but otherwise good or do you mean only a little of the sound was getting through (i.e. broken up)?

**Reply by Schmidl83** [reply]
Thanks for your quick reply.
The sound, e.g. guitars, was very quiet and the vocals sounded very dull. 
So it wasn't broken up or anything, the connection  in regards to latency was very good.

**Reply by pljones** [reply]
I can see you have "Mono" selected in the "Audio/Network Setup" tab - but have two separate input channels.

Are you using both inputs (e.g. guitar on one, mic on the other, down mixed to a single mono channel)?

Can you also check the settings on the "Advanced Setup" settings tab for the "Input Balance".  The "Input Balance" adjust the mix between the two inputs.

What I can't think of is why this would start happening to a group of people at the same time due to something in Jamulus.
